### PR TITLE
Develop minimal partner service

### PR DESCRIPTION
**Pull request by hackebrot**
The service is built in Python with FastAPI and can be run inside Docker.

### CODE REVIEW

**Comment by ashrivastava-qa** [comment]
LGTM 👍

**Review by ashrivastava-qa — APPROVED** [review]

File: partner/app/main.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+class Tile(BaseModel):
+    id: int
+    name: str
+    click_url: str
+    image_url: str
+    impression_url: str
+    advertiser_url: str
+
+
+TILES = [
```
Comment (jrconlin) [inline partner/app/main.py:25]: change request: The contile filter validation system attempts to parse these as valid URLs before checking the host. We may want to use something that is more parseable (e.g. `https://example.com/click_url`) As these are, they would never pass Contile filtering.

File: partner/app/main.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    sub1: str = Query(..., example="123456789"),
+    sub2: str = Query(..., example="placement1"),
+    country_code: str = Query(..., alias="country-code", example="US"),
+    region_code: str = Query(..., alias="region-code", example="NY"),
+    form_factor: str = Query(..., alias="form-factor", example="desktop"),
+    os_family: str = Query(..., alias="os-family", example="macOS"),
+    v: str = Query(..., example="1.0"),
+    out: str = Query("json", example="json"),
+    results: int = Query(1, example=2),
+):
```
Comment (jrconlin) [inline partner/app/main.py:74]: I know that this is just a simple responder, but do you want to add some validation here to ensure that we're seeing all the required values being passed?

**Review by jrconlin — CHANGES_REQUESTED** [review]

File: partner/app/main.py (hunk 2 above)
Comment (hackebrot) [reply partner/app/main.py:74]: The `...` value in ` Query(..., example="1.0")` makes the Query parameter required. Missing at least one of the required parameters results in a `422`. The parameters `out` and `results` are optional according to the API Spec PDF.

File: partner/app/main.py (hunk 1 above)
Comment (hackebrot) [reply partner/app/main.py:25]: That makes sense! I'll update that. Thanks!

**Review by jrconlin — APPROVED** [review]
👍🏻

**Pull request merged**